source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { match as Match } from 'react-router';
+import { compose } from 'recompose';
+import { withRouter } from 'react-router';
+import { Switch, Route } from 'react-router-dom';
+
+import { requireAuth } from '@lib/auth';
+import { NotFound } from '@ui/routes/errors';
+import { withLayout } from '@ui/components/layout';
+
+import MyProjectsTable from './my-projects-table';
+import OrgProjectsTable from './org-projects-table';
+import ArchivedProjectsTable from './archived-projects-table';
+
+import Header from './header';
+
+export const pathName = '/projects';
+
+class ProjectsRoot extends React.Component {
+
+  render() {
+
+    return (
+      <div className='ui container'>
+        <Switch>
+          <Route exact path='/projects/own' render={(routeProps) => (
```

Comment:
to continue with the pattern used everywhere else, component should be used instead of render. it keeps our routing concerns small and focused :)

currently, each of the 3 tables are a component / index.tsx file -- so maybe that could change to: 
```
/whatever-this-path-is/projects/
  - {own, my-projects, organization-projects} /
    - index.tsx (the route config that would do exactly the same as each of these render blocks)
    - table.tsx (the existing index.tsxs)

```